packages/cli-tools/src/cacheManager.ts

```diff
@@ -48,6 +49,23 @@ function getCacheRootPath() {
   return cachePath;
 }
 
+function removeProjectCache(name: string) {
+  const cacheRootPath = getCacheRootPath();
+  try {
+    const fullPath = path.resolve(cacheRootPath, name);
```

Comment:
nit: we can move `fullPath` outside try catch and re-use it in `catch` statement.